docs/changes/0.24.0.rst

```diff
@@ -0,0 +1,21 @@
+Changelog for QCoDeS 0.24.0
+===========================
+
+The March 2021 release of QCoDeS.
+
+---------
+Improved:
+---------
+
+- Support export to xarray for data with non unique indices (#2848).
+- Update use of Slack API in qcodes.utils.slack.Slack class (#2813)
+- Support dataset export to NetCDF or CSV after measurement ends (#2782)
+- Export more metadata to xarray dataset (#2854)
+
+
+-----------------
+Improved drivers:
+-----------------
+
+- Remove hardcoded acquire mode and allow users to set acquire mode for trace on Infiniium driver (#2837) *Note: Because the mode of the instrument is no longer set in the constructor, this is a breaking change.*
```

Comment:
for breaking changes, if they are present, we usually add a separate section (see previous changelogs), this makes it more visible to the users.

Reply:
yes, indeed, like that. however we usually put the breaking changes at the top for visibility :)